aiobotocore/paginate.py

```diff
@@ -59,12 +59,14 @@ def _init_pager(self):
         self._starting_truncation = 0
         self._inject_starting_params(self._current_kwargs)
 
-    @asyncio.coroutine
-    def next_page(self):
+    async def __aiter__(self):
+        return self
+
+    async def __anext__(self):
```

Comment:
Lets do this backward compatible for now, since my  tests working only for 3.4,  then we can think about moving to 3.5.